### PR TITLE
ESM landing page in french

### PR DESCRIPTION
**Pull request by caldav**
## Done

* ESM landing page in French
* [Copy doc](https://docs.google.com/document/d/14owLexmd5yn9g9Opq2oYAw-oxG2k2YKLgEi9bGqzzMg/edit)

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Ensure the page at https://www-ubuntu-com-canonical-websites-pr-4623.run.demo.haus/engage/fr/esm looks right and matches the [copy doc](https://docs.google.com/document/d/14owLexmd5yn9g9Opq2oYAw-oxG2k2YKLgEi9bGqzzMg/edit)

### CODE REVIEW

**Comment by anasereijo** [comment]
Hi @caldav :)

- Any reason why the contact form is down on the page and not on the right side as we usually do?
- The image is too close to the title can you make it `class="col-4 u-hide--small u-align--center`?

**Comment by caldav** [comment]
@anasereijo 
* yes, it's a fairly long article - sort of a replacement for our lack of french blog. And I do believe that in this case, the content needs to be read for the form to have any kind of effectiveness. That's why the form is at the end and not on the side.
* Looks much better indeed!

File: templates/engage/fr/esm.html [hunk 1]
```diff
@@ -0,0 +1,119 @@
+{% extends "engage/base_engage.html" %}
+
+{% block title %}Canonical annonce l’Extension Sécurité et Maintenance pour Ubuntu 14.04.{% endblock %}
+{% block meta_description %}Sécurité, conformité, RGPD: gardez vos serveurs en sécurité après la fin de support officielle d'Ubuntu 14.04 LTS en avril 2019.{% endblock %}
+{% block meta_copydoc %}https://docs.google.com/document/d/14owLexmd5yn9g9Opq2oYAw-oxG2k2YKLgEi9bGqzzMg/edit{% endblock meta_copydoc %}
+
+
+{% block content %}
+
+<section class="p-strip is-deep p-takeover--private-cloud-economics">
```
Comment (pmahnke) [inline templates/engage/fr/esm.html]: can you add lang="fr" for the containers in french?

Comment (caldav) [reply templates/engage/fr/esm.html]: Very good point, also added to the other fr landing page

**Comment by anasereijo** [comment]
@caldav thanks for making the changes, looks good 👍

**Pull request merged**